psi4/share/psi4/basis/def2-qzvpp-ri.gbs

```diff
@@ -718,7 +718,7 @@ S  1  1.00
 S  1  1.00
  0.0665549  1.0
 P  1  1.00
- 217       1.0
+ 217.0     1.0
 P  1  1.00
```

Comment:
We should fix the the parser in the the next pass through the py-side basis set machinery.  I also modified this file for the ECP code, so will be on the lookout for similar issues.